packages/generator-langium/templates/packages/language/src/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './<%= language-id %>-module.js';
+export * from './<%= language-id %>-validator.js';
+export * from './generated/ast.js';
+export * from './generated/grammar.js';
+export * from './generated/module.js';
+export { default as monarchSyntax } from './syntaxes/<%= language-id %>.monarch.js';
```

Comment:
Question: We should only conditionally generate this export - if the `web` option is selected.

Reply:
This was a bit more work, because I needed to change the workflow, but it looks now, IMO.